packages/test/test-end-to-end-tests/src/test/migration-shim/dataMigration.spec.ts

```diff
@@ -98,24 +97,18 @@ class TestDataObject extends DataObject {
 	}
 }
 
-const builder = new SchemaBuilder({ scope: "test" });
+const builder = new SchemaFactory("test");
 // For now this is the schema of the view.root
-const inventorySchema = builder.object("abcInventory", {
+class InventorySchema extends builder.object("abcInventory", {
 	quantity: builder.number,
-});
-
-// This is some schema to be updated later
-const inventoryFieldSchema = SchemaBuilder.required(inventorySchema);
-const schema = builder.intoSchema(inventoryFieldSchema);
+}) {}
```

Comment:
Any particular reason to extend here, vs. `const InventorySchema = builder.object...`?  Not sure I have any preference, just curious.